src/nc/gui/MainWindow.cpp

```diff
@@ -88,7 +88,8 @@ MainWindow::MainWindow(Branding branding, QWidget *parent):
 
     logToken_ = LogToken(logger);
 
-    settings_ = new QSettings(branding_.organizationName(), branding_.applicationName(), this);
+    settings_ = new QSettings(QString(branding.applicationName() + ".ini"), QSettings::IniFormat, this);
```

Comment:
Yea, that's correct. But why wouldn't the current directory be readable? Anyways, I can revert the commit without problems (doesn't matter to me). Do you agree with the rest of the changes?